package/utils/wtmcu/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wtmcu
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.com/evgenyz/$(PKG_NAME)/-/archive/v$(PKG_VERSION)
+PKG_HASH:=557f8b6a9771dcee094692da4480274695ec2af53871a3ddea069ee94e207509
+
+PKG_LICENSE:=BSD-2-Clause
+PKG_MAINTAINER:=Evgeny Kolesnikov <evgenyz@gmail.com>
+
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-v$(PKG_VERSION)
```

Comment:
If this becomes a local package, this line can be dropped, so we use the default.